src/about.html

```diff
@@ -0,0 +1,411 @@
+                  give you a personal look into the personalities of the chefs
+                  and the philosophies behind their culinary art."
+                </p>
+                <div
+                  class="col carousel-img container d-flex justify-content-center align-items-center"
+                >
+                  <img
+                    class="rounded-5 img-fluid carousel-img-cont"
+                    src="../assets/imgs/jonas-kakaroto-mjRwhvqEC0U-unsplash.jpg"
+                    alt=""
+                  />
+                  <p class="ms-3 text-danger">Michael B.</p>
+                </div>
+              </div>
+            </div>
+
+            <div class="carousel-item">
+              <div class="d-block w-100 row carousel-about">
+                <p
+                  class="text-center text-dark col fw-semibold carousel-txt w-auto"
+                >
+                  "Mama's kitchen is an amazing app with loads of interesting
+                  stories about chefs from around the world. You get to see
+                  "why" the chefs cook the way they do, "how" they cook and what
+                  joy they get from cooking. The quality of the videos is simply
+                  amazing! My wife and I love to have a glass of wine and watch
+                  a few videos each week!"
+                </p>
+                <div
+                  class="col carousel-img container d-flex justify-content-center align-items-center"
+                >
+                  <img
+                    class="rounded-5 img-fluid carousel-img-cont"
+                    src="../assets/imgs//kelly-sikkema-JN0SUcTOig0-unsplash.jpg"
+                    alt=""
+                  />
+                  <p class="ms-3 text-danger">Vanessa L.</p>
+                </div>
+              </div>
+            </div>
+
+            <div class="carousel-item">
+              <div class="d-block w-100 row carousel-about">
+                <p
+                  class="text-center text-dark fw-semibold col carousel-txt w-auto"
+                >
+                  "I really enjoy the way that this site is set up. The
+                  introductory biography is excellent and gives you the context
+                  to understand the chefs' personalities and what drives them.
+                  The classes are wonderful and techniques are explained well.
+                  This is not a quickie family dinner kind of site. It is for
+                  the explorer of flavor and joy of learning."
+                </p>
+                <div
+                  class="col carousel-img container d-flex justify-content-center align-items-center"
+                >
+                  <img
+                    class="rounded-5 img-fluid carousel-img-cont"
+                    src="../assets/imgs/michael-austin-jgSAuqMmJUE-unsplash.jpg"
+                    alt=""
+                  />
```

Comment:
- [ ] kindly consider making sure that this section looks exactly like the one given in the design on Behance to meet the project design requirement and remember you are only allowed to change the content but the design remains the same 

- your design 👇 
<img width="370" alt="Screen Shot 2023-05-23 at 17 13 17" src="https://github.com/Yonashenok/first-capstone-project/assets/60146030/a92bec45-4128-45e8-9efb-9f7afb36447b">

- provided design 👇 
<img width="447" alt="Screen Shot 2023-05-23 at 17 13 57" src="https://github.com/Yonashenok/first-capstone-project/assets/60146030/3ef9ed8c-eb9a-46b3-9b8f-40bd3d3902d9">

- [ ] While fixing that make sure that on both screen sizes, the design remains the same exactly as the one on Behance
either the way cards are displayed the design the background color the texts on the card like everything to deliver whats expected